autoload/javacomplete.vim

```diff
@@ -121,7 +113,7 @@ function! javacomplete#GetBase(extra)
 endfunction
 
 function! s:GetBase(extra)
-  let base = expand("~". g:FILE_SEP. ".javacomplete2". g:FILE_SEP. a:extra)
+  let base = expand(g:JavaComplete_BaseDir. g:FILE_SEP. a:extra)
```

Comment:
That's ok.

On 01-06 (21:43), Wang Shidong wrote:
>    In [1]autoload/javacomplete.vim:
> 
>  > @@ -121,7 +113,7 @@ function! javacomplete#GetBase(extra)
>  >  endfunction
>  >
>  >  function! s:GetBase(extra)
>  > -  let base = expand("~". g:FILE_SEP. ".javacomplete2". g:FILE_SEP. a:extra)
>  > +  let base = expand(g:JavaComplete_BaseDir. g:FILE_SEP. a:extra)
> 
>    oh sorry,I will do it tonight,now i am on the bus
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/artur-shaik/vim-javacomplete2/pull/176#discussion_r49043709
>    2. https://github.com/artur-shaik/vim-javacomplete2/pull/176/files#r49043709
>    3. https://github.com/artur-shaik/vim-javacomplete2/pull/176/files#r49043709

-- 
Best regards,
Artur Shaikhullin